docs/reference/accessibility/orca-preferences/index.md

```diff
@@ -0,0 +1,65 @@
+(orca-preferences)=
+# Orca preferences
+
+Orca preferences allow you to customize functionality in Orca which applies to all applications. An example of an Orca preference is key echo because key echo is something that applies to all applications.
+
+Note that Orca preferences can be customized on an application-by-application basis. For instance you can set the default key echo to words and then set the key echo for Pidgin to be none. Having done so, Orca would always echo each word that you typed, unless you were in Pidgin. 
+
+Orca preferences can be configured globally or on the application level. Application-level preferences allow you to customize Orca functionality that only applies in certain environments, such as on web pages or in chat applications. As a result, you will only find these options available in the application-specific preferences dialogs and only for those applications to which these options apply. 
```

Comment:
This paragraph feels like it rephrases the previous one to a degree. Maybe this could be consolidated.